Resources/Private/Less/modules/score.less

```diff
@@ -0,0 +1,111 @@
+/*
+ *
+ * Score
+ * ================================================
+ * Specials for the Score view
+ *
+ * Author: Thomas Jung <thomas@tjwd.de>
+ * License: All rights reserved
```

Comment:
This license is not compatible with GPL-3.0, so the code cannot be used for the DFG viewer.

Reply:
See for example this discussion about "All rights reserved" in the context of the GPL:
https://lists.debian.org/debian-legal/2007/06/msg00249.html.

Reply:
You are right!

@gnuj Can we do something about this? May I put all the files under GPL license?

Reply:
@sebastian-meyer Yes, of course! No problem on my side if the licence lines are removed. These are very old rudiments, and I think the same applies to the EOF statements in the last line. I would be happy to look into this in a separate branch in the course of the coming week. Thats okay? 🤔

Reply:
Good idea.